pkgs/development/python-modules/coqui-trainer/conqui-trainer.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/requirements.txt b/requirements.txt
```

Comment:
Like @mweinelt, I generally prefer sed/substituteInPlace/etc instead of checked-in patch files. Patch files are much harder to update once a new version comes around. Using sed/substituteInPlace usually has a longer shelf-life.

But more importantly this patch file is no longer necessary since we can `fetchpatch` the upstream PR https://github.com/coqui-ai/Trainer/pull/21.